# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sox/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- sox-14.0.0/src/wav.c.vanilla	2007-09-14 23:22:39.000000000 +0200
+++ sox-14.0.0/src/wav.c	2007-09-14 23:22:59.000000000 +0200
@@ -24,7 +24,7 @@
 #include "ima_rw.h"
 #include "adpcm.h"
 #ifdef EXTERNAL_GSM
-#include <gsm/gsm.h>
+#include <gsm.h>
 #else
 #include "../libgsm/gsm.h"
 #endif
--- sox-14.0.0/src/gsm.c.vanilla	2007-09-14 23:22:27.000000000 +0200
+++ sox-14.0.0/src/gsm.c	2007-09-14 23:22:33.000000000 +0200
@@ -29,7 +29,7 @@
 #include "sox_i.h"
 
 #ifdef EXTERNAL_GSM
-#include <gsm/gsm.h>
+#include <gsm.h>
 #else
 #include "../libgsm/gsm.h"
 #endif
